config/features/config.go

```diff
@@ -186,6 +186,7 @@ func ConfigureBeaconChain(ctx *cli.Context) {
 		cfg.EnableBatchGossipAggregation = true
 	}
 	Init(cfg)
+	return nil
```

Comment:
What is the benefit of changing the function signature to return an error, we dont return it anywhere here.

Reply:
I think there was a reason that got refactored away, I'll clean it up.

Reply:
I modified `configureTestnet` to return an error upon conflict (from `SetActive`), which brought back the requirement for this method to return an error.